app/models/workarea/pricing/calculators/taxjar_tax_calculator.rb

```diff
@@ -2,10 +2,9 @@ module Workarea
   module Pricing
     module Calculators
       # calculates order/shipping sales tax via the Taxjar API
-      class TaxjarTaxCalculator
-        include Calculator
-
+      class TaxjarTaxCalculator < TaxCalculator
```

Comment:
This (and the `return super` down below) was the key. Since the TaxCalculator is swapped out in initializers, I figured just inheriting from it and calling super was the best way to prevent TaxJar stuff from being calculated in tests. It should only call TaxJar when it needs to in very specific (not yet written) tests.